src/d_loop.c

```diff
@@ -755,20 +755,7 @@ void TryRunTics (void)
 
     if (new_sync)
     {
-        if (uncapped)
-        {
-            // decide how many tics to run
-            if (realtics < availabletics-1)
-                counts = realtics+1;
-            else if (realtics < availabletics)
-                counts = realtics;
-            else
-                counts = availabletics;
-        }
```

Comment:
I tried to fix the newsync uncapped mode, but didn't get a good result. I think we should remove this, it is definitely not working properly.
We can just force 35 FPS in newsync for now. I'll try the rollback/prediction after the DRS and menus are complete.

Reply:
Alright, I was about to ask this. Thank you.